specification/cognitiveservices/data-plane/TextAnalytics/v2.0/TextAnalytics.json

```diff
@@ -0,0 +1,548 @@
+      }
+     }
+    },
+    "deprecated": false
+   }
+  },
+  "/v2.0/sentiment": {
+   "post": {
+    "tags": [
+     "Sentiment"
+    ],
+    "summary": "The API returns a numeric score between 0 and 1. \r\n            Scores close to 1 indicate positive sentiment, while scores close to 0 indicate negative sentiment. \r\n            Sentiment score is generated using classification techniques. \r\n            The input features to the classifier include n-grams, features generated from part-of-speech tags, and word embeddings. \r\n            See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview#supported-languages\">Text Analytics Documentation</a> for details about the languages that are supported by sentiment analysis.",
+    "operationId": "Sentiment",
+    "consumes": [
+     "application/json"
+    ],
+    "produces": [
+     "application/json"
+    ],
+    "parameters": [
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     },
+     {
+      "name": "input",
+      "in": "body",
+      "description": "Collection of documents to analyze.",
+      "required": true,
+      "schema": {
+       "$ref": "#/definitions/MultiLanguageBatchInputV2"
+      }
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call results in each valid document getting a sentiment score between 0 and 1",
+      "schema": {
+       "$ref": "#/definitions/SentimentBatchResultV2"
+      },
+      "examples": {
+       "application/json": {
+        "documents": [
+         {
+          "score": 0.92,
+          "id": "1"
+         },
+         {
+          "score": 0.85,
+          "id": "2"
+         },
+         {
+          "score": 0.34,
+          "id": "3"
+         }
+        ],
+        "errors": null
+       }
+      }
+     },
+     "default": {
+      "description": "Error Response",
+      "schema": {
+       "$ref": "#/definitions/ErrorResponse"
+      }
+     }
+    },
+    "deprecated": false
+   }
+  }
+ },
+ "definitions": {
+  "MultiLanguageBatchInputV2": {
+   "type": "object",
+   "properties": {
+    "documents": {
+     "type": "array",
+     "items": {
+      "$ref": "#/definitions/MultiLanguageInputV2"
+     }
+    }
+   },
+   "example": {
+    "documents": [
+     {
+      "language": "en",
+      "id": "1",
+      "text": "Hello world. This is some input text that I love."
+     },
+     {
+      "language": "fr",
+      "id": "2",
+      "text": "Bonjour tout le monde"
+     },
+     {
+      "language": "es",
+      "id": "3",
+      "text": "La carretera estaba atascada. Había mucho tráfico el día de ayer."
+     }
+    ]
+   }
+  },
+  "MultiLanguageInputV2": {
+   "type": "object",
+   "properties": {
+    "language": {
+     "description": "This is the 2 letter ISO 639-1 representation of a language.\r\n            For example, use \"en\" for English; \"es\" for Spanish etc.,",
+     "type": "string"
+    },
+    "id": {
```

Comment:
Is id read-only?

Reply:
No, the "id" is user-defined.

Reply:
Oh nevermind, the return value is readonly.